src/EPR.Calculator.API/Builder/Summary/ThreeSa/ThreeSaCostsSummary.cs

```diff
@@ -0,0 +1,42 @@
+﻿using EPR.Calculator.API.Builder.Summary.SaSetupCosts;
+using EPR.Calculator.API.Builder.Summary.ThreeSa;
+using EPR.Calculator.API.Constants;
+using EPR.Calculator.API.Models;
+
+namespace EPR.Calculator.API.Builder.Summary.ThreeSA
+{
+    public static class ThreeSaCostsSummary
+    {
+        public static readonly int ColumnIndex = 210;
+
+        public static IEnumerable<CalcResultSummaryHeader> GetHeaders()
+        {
+            return [
+                new CalcResultSummaryHeader { Name = $"{ThreeSaCostHeader.SaOperatingCostsWithoutBadDebtProvisionTitleSection3}", ColumnIndex = ColumnIndex },
+                new CalcResultSummaryHeader { Name = $"{ThreeSaCostHeader.BadDebtProvisionTitleSection3}", ColumnIndex = ColumnIndex + 1 },
+                new CalcResultSummaryHeader { Name = $"{ThreeSaCostHeader.SaOperatingCostsWithBadDebtProvisionTitleSection3}", ColumnIndex = ColumnIndex + 2 }
+            ];
+        }
+
+        public static decimal GetThreeSaCostsWithoutBadDebtProvision(CalcResult calcResult)
+        {
+            return calcResult.CalcResultParameterOtherCost.SaOperatingCost.OrderByDescending(t => t.OrderId).FirstOrDefault().TotalValue;
```

Comment:
Please use First if you are expecting it to always have a value.
If using FirstOrDefault then you need to use ?.Totalvalue and change the return type from decimal to Nullable<decimal>